src/MongoDB.Driver.Core/Core/Clusters/MultiServerCluster.cs

```diff
@@ -38,7 +38,6 @@ internal sealed class MultiServerCluster : Cluster, IDnsMonitoringCluster
         private Thread _dnsMonitorThread;
         private readonly CancellationTokenSource _monitorServersCancellationTokenSource;
         private volatile ElectionInfo _maxElectionInfo;
-        private Task _monitorServersTask;
         private volatile string _replicaSetName;
         private readonly AsyncQueue<ServerDescriptionChangedEventArgs> _serverDescriptionChangedQueue;
```

Comment:
It looks like this property is not used anymore?

Reply:
sorry, I meant: `_serverDescriptionChangedQueue`

Reply:
You are correct! Fixed!